test/application/bot.test.js.disabled

```diff
@@ -1,3 +1,7 @@
+/**
+ * Disabled due to issues with Discord.JS v14.
+ * TODO: rework this so it will work with Discord.JS v14!
```

Comment:
Not a comment for change, just an FYI. My analysis is that the test is doing 'async work' after the Client is constructed. Bad practice? Yes. Reality? Also yes.

This is a problem we could solve by mocking. It's not easy and takes quite some Jest digging. Your comment is 100% justified here.

Reply:
I think we should create an issue for this, so we won’t foget!